diff --git a/cmake/MKL.cmake b/cmake/MKL.cmake
index 57acaab3..77e6a076 100644
--- a/cmake/MKL.cmake
+++ b/cmake/MKL.cmake
@@ -30,8 +30,6 @@ function(detect_mkl LIBNAME)
 
     message(STATUS "Detecting Intel(R) MKL: trying ${LIBNAME}")
 
-    find_path(MKLINC mkl_cblas.h
-        HINTS ${MKLROOT}/include $ENV{MKLROOT}/include)
     if(NOT MKLINC)
         file(GLOB_RECURSE MKLINC
                 ${CMAKE_CURRENT_SOURCE_DIR}/external/*/mkl_cblas.h)
